.gitignore

```diff
@@ -1,4 +1,369 @@
+
+# Created by https://www.toptal.com/developers/gitignore/api/visualstudio,dotnetcore
+# Edit at https://www.toptal.com/developers/gitignore?templates=visualstudio,dotnetcore
+
+### DotnetCore ###
+# .NET Core build folders
 bin/
 obj/
-/packages/
-docs/
```

Comment:
Those two lines we still need.
They are related tot eh GitHub pages deployment and the setup for that.

Reply:
readded `docs/`
`/packages/` not needed as it's already included in the template as `**[Pp]ackages/*`